pybird_dev/greenfunction.py

```diff
@@ -1,17 +1,24 @@
+import numpy as np
 from scipy.integrate import quad
 from scipy.special import hyp2f1
+from scipy.integrate import odeint
+
 
 class GreenFunction(object):
-    
-    def __init__(self, Omega0_m, w=None, quintessence=False):
+
+    def __init__(self, Omega0_m, w=None, quintessence=False, Omega_rc=None, MG=False):
         self.Omega0_m = Omega0_m
         self.OmegaL_by_Omega_m = (1.-self.Omega0_m)/self.Omega0_m
         self.wcdm = False
         self.quintessence = False
-        if w is not None: 
+        self.MG = False
+        if w is not None:
             self.w = w
             if quintessence: self.quintessence = True
             else: self.wcdm = True
+        if Omega_rc is not None: #should be elif since don't want both wcDM and nDGP?
```

Comment:
In principle it's possible to have a model in which there are both quintessence and nDGP

Reply:
Ok, do we want to include this possibility now [if so I will add the wCDM behaviour as an "if wcdm:" option in the H_NC () and dHda_NC() functions], or just keep the background fixed to LCDM in the nDGP case for the moment?

Reply:
This is related to the discussion in the comment about lines 8 and 9.

Reply:
Ok, let's keep this as it is and then we'll think about more general cases as they come.